bertopic/_bertopic.py

```diff
@@ -3980,18 +3980,29 @@ def _extract_topics(
             embeddings: The document embeddings
             mappings: The mappings from topic to word
             verbose: Whether to log the process of extracting topics
+            calculate_representation: Whether to extract the topic representations
 
         Returns:
             c_tf_idf: The resulting matrix giving a value (importance score) for each word per topic
         """
         if verbose:
-            logger.info("Representation - Extracting topics from clusters using representation models.")
+            action = "Representation" if calculate_representation else "Topics"
+            logger.info(
+                f"{action} - Extracting topics from clusters{' using representation models' if calculate_representation else ''}."
+            )
+
         documents_per_topic = documents.groupby(["Topic"], as_index=False).agg({"Document": " ".join})
         self.c_tf_idf_, words = self._c_tf_idf(documents_per_topic)
-        self.topic_representations_ = self._extract_words_per_topic(words, documents)
+        self.topic_representations_ = self._extract_words_per_topic(
+            words,
+            documents,
+            calculate_representation=calculate_representation,
+            calculate_aspects=calculate_representation,
```

Comment:
Just to be sure I understand this correctly. You added this here because when you only calculate the base representation  (`calculate_representation=False`) you do not want the additional representations (`calculate_aspects=False`), right?

If so, we might need to think about the term `calculate_representation` because it should refer to fine-tuning representations instead right? Or did you name it did way because technically, it might also be the default representation when you set it as `"Main"`?

Reply:
Thats right.

Indeed is about fine tuning. But yea, the nomenclature here is confusing for me as well.

It could be changed to "fine_tune_representations"

I will gather my thoughts on one single comment bellow, since this touches on the other comments.